### PR TITLE
[main][feat] mm flat slop model

### PR DESCRIPTION
**Pull request by jr-alpaca**

### CODE REVIEW

File: solidity/contracts/money-market/interest-models/MMFlatSlopeModel1.sol [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  uint256 public constant CEIL_SLOPE_2 = 100e18;
+  uint256 public constant CEIL_SLOPE_3 = 100e18;
+
+  uint256 public constant MAX_INTEREST_SLOPE_1 = 6e16;
+  uint256 public constant MAX_INTEREST_SLOPE_2 = 6e16;
+  uint256 public constant MAX_INTEREST_SLOPE_3 = 6e16;
+
+  /// @dev Return the interest rate per second, using 1e18 as denom.
+  function getInterestRate(uint256 debt, uint256 floating) external pure returns (uint256) {
+    if (debt == 0 && floating == 0) return 0;
```
Comment (alpanaca) [inline solidity/contracts/money-market/interest-models/MMFlatSlopeModel1.sol:18]: should be || ?

Comment (alpanaca) [reply solidity/contracts/money-market/interest-models/MMFlatSlopeModel1.sol:18]: Just a question of how do you want to handle when debt == 0 and floating != 0

Comment (jr-alpaca) [reply solidity/contracts/money-market/interest-models/MMFlatSlopeModel1.sol:18]: cannot do ||. If debt is max out it should not return 0. I think if statement can be removed. Just leave it there to follow the previous model.

Comment (alpanaca) [reply solidity/contracts/money-market/interest-models/MMFlatSlopeModel1.sol:18]: I think we can just remove it and return constant value.

File: solidity/tests/money-market/interest-models/MMFlatSlope1.t.sol [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  function setUp() external {
+    _flatSlopeModel1 = new MMFlatSlopeModel1();
+  }
+
+  function _findInterestPerYear(uint256 _interestPerSec) internal pure returns (uint256) {
+    return _interestPerSec * 365 days;
+  }
+
+  function testCorrectness_getInterestRate() external {
```
Comment (alpanaca) [inline solidity/tests/money-market/interest-models/MMFlatSlope1.t.sol:21]: can just to fuzz test.

**Review by 69alpacas — DISMISSED** [review]

File: solidity/contracts/money-market/interest-models/MMFlatSlopeModel1.sol [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+pragma solidity 0.8.19;
+
+contract MMFlatSlopeModel1 {
+  /// @dev Return the interest rate per second, using 1e18 as denom.
+  function getInterestRate(
+    uint256, /*debt*/
+    uint256 /*floating*/
+  ) external pure returns (uint256) {
+    // 6% flat rate
+    return 60000000000000000;
```
Comment (alpanaca) [inline solidity/contracts/money-market/interest-models/MMFlatSlopeModel1.sol:11]: Shoudn't it be 60000000000000000 / 365 days => ??

**Review by alpanaca — APPROVED** [review]

**Pull request merged**